sbnana/SBNAna/ananue/PAC2020/SBND/helper_eff_spill.h

```diff
@@ -77,7 +77,6 @@ struct PlotDef {
 };
 
 const Binning kBDTBinning = Binning::Simple(40, 0, 1.0);
-const Binning kEnergyBinning = Binning::Simple(40, 0, 3000);
 const Binning kEnergyBinningGeV = Binning::Simple(30, 0, 3);
 const Binning kdEdxBinning = Binning::Simple(40, 0, 10);
 const Binning kGapBinning = Binning::Simple(20, 0, 10);
```

Comment:
Here on the binning there's:
```
const Binning kTimeBinning = Binning::Simple(70, -3500, 3500);
```
which I believe it's still on `ns`

Reply:
I am pretty sure that is `us` to span the TPC readout time

Reply:
Oh ok, I thought it was the spill time.